### PR TITLE
Set/Get the netports dhcp ipv6 lease time and lease time remaining

### PR DESCRIPTION
**Pull request by metelik**

### CODE REVIEW

File: src/bacnet/basic/object/netport.c [hunk 1]
```diff
@@ -2665,6 +2669,88 @@ bool Network_Port_IPv6_DHCP_Server_Set(
...
+ *
+ * @return IPv6_DHCP_Lease_Time
+ */
+uint32_t Network_Port_IPv6_DHCP_Lease_Time(uint32_t object_instance)
+{
+    const unsigned index = Network_Port_Instance_To_Index(object_instance);
+    uint32_t value = 0;
+
+    if (index < BACNET_NETWORK_PORTS_MAX) {
+        if (Object_List[index].Network_Type == PORT_TYPE_BIP6) {
```
Comment (PeterJeaiem) [inline src/bacnet/basic/object/netport.c:2686]: can be put in the same if statement 
    if ((index < BACNET_NETWORK_PORTS_MAX) &&  (Object_List[index].Network_Type == PORT_TYPE_BIP6) )

File: src/bacnet/basic/object/netport.c [hunk 2]
```diff
@@ -2665,6 +2669,88 @@ bool Network_Port_IPv6_DHCP_Server_Set(
...
+ * @return  true if ipv6-dhcp-lease-time was set
+ */
+bool Network_Port_IPv6_DHCP_Lease_Time_Set(
+    uint32_t object_instance, const uint32_t value)
+{
+    bool status = false;
+    unsigned index = Network_Port_Instance_To_Index(object_instance);
+
+    if (index < BACNET_NETWORK_PORTS_MAX) {
+        if (Object_List[index].Network_Type == PORT_TYPE_BIP6) {
```
Comment (PeterJeaiem) [inline src/bacnet/basic/object/netport.c:2710]: same here

File: src/bacnet/basic/object/netport.c [hunk 3]
```diff
@@ -2665,6 +2669,88 @@ bool Network_Port_IPv6_DHCP_Server_Set(
...
+ *
+ * @return IPv6_DHCP_Lease_Time
+ */
+uint32_t Network_Port_IPv6_DHCP_Lease_Time_Start(uint32_t object_instance)
+{
+    const unsigned index = Network_Port_Instance_To_Index(object_instance);
+    uint32_t value = 0;
+
+    if (index < BACNET_NETWORK_PORTS_MAX) {
+        if (Object_List[index].Network_Type == PORT_TYPE_BIP6) {
```
Comment (PeterJeaiem) [inline src/bacnet/basic/object/netport.c:2746]: same

**Review by PeterJeaiem — APPROVED** [review]

**Review by dcmaverick — APPROVED** [review]

**Review by alanj853 — APPROVED** [review]
Approved, but agree with Peter's suggestions.

**Pull request merged**